storage/Devices/Disk.cc

```diff
@@ -133,7 +133,7 @@ namespace storage
     {
 	Disk* disk = dynamic_cast<Disk*>(device);
 	if (!disk)
-	    throw DeviceHasWrongType("device is not a Disk");
+	    ST_THROW(DeviceHasWrongType("device is not a Disk"));
```

Comment:
Yes, that is used often. I also realized. But sometimes the exception is of type DeviceHasWrongType and sometimes of HolderHasWrongType. My idea it to use a templated function that replaces the complete function body.